sql/tablewriter.go

```diff
@@ -271,6 +271,9 @@ func (tu *tableUpserter) init(txn *client.Txn) error {
 		if err != nil {
 			return err
 		}
+		// The fetchCols can also contain columns undergoing mutation; update
+		// requestCols.
+		requestedCols = tu.ru.fetchCols
```

Comment:
Ah, then add a fetchCols to tableUpserter and set it in both branches. In all of the "simple" Insert, Update, Delete tableWriters and rowWriters, the "requestedCols" is used as an immutable input and the "fetchCols" handed back is used to select what is fetched. I'd like to keep this one consistent